src/__tests__/proposals.int.test.ts

```diff
@@ -78,6 +78,103 @@ describe('/proposals', () => {
         );
     });
 
+    it('returns according to pagination parameters', async () => {
+      await db.query(`
+        INSERT INTO opportunities (
+          title,
+          created_at
+        )
+        VALUES
+          ( '🔥', '2525-01-02T00:00:01Z' )
+      `);
+      await db.query(`
+        INSERT INTO applicants (
+          external_id,
+          opted_in,
+          created_at
+        )
+        VALUES
+          ( '12345', 'true', '2022-07-20 12:00:00+0000' ),
+          ( '67890', 'false', '2022-07-20 12:00:00+0000' );
+      `);
```

Comment:
Likewise:

```suggestion
      await db.sql('applicants.insertOne', { externalId: '12345', optedIn: true });
      await db.sql('applicants.insertOne', { externalId: '67890', optedIn: false });
```

Oh and do we need both? I only see references to the first in the test below.